index.html

```diff
@@ -0,0 +1,177 @@
+            hestiate to contact me.</p>
+
+        <div class="Right contact">
+            <a class="media-text">LET’S CONNECT</a>
+
+            <ul class="right-icons">
+                <li><a href="https://twitter.com/Ernestbore3" target="_blank" class="media-link" rel="noopener"><i
+                            class="fab fa-twitter"></i></a></li>
+                <li><a href="https://www.linkedin.com/in/ernest-bore-3b6042211/" target="_blank" class="media-link"
+                        rel="noopener"><i class="fab fa-linkedin"></i></a></li>
+                <li><a href="https://medium.com/" target="_blank" class="media-link" rel="noopener"><i
+                            class="fab fa-medium-m"></i></a></li>
+                <li><a href="https://github.com/ernestkorir" target="_blank" class="media-link" rel="noopener"><i
+                            class="fab fa-github"></i></a></li>
+                <li><a href="https://angel.co/ernestkorir" target="_blank" class="media-link" rel="noopener"><i
+                            class="fab fa-angellist"></i></a></li>
+            </ul>
+        </div>
+    </section>
+    <section class="work">
+        <div class="card-work-1">
+            <div class="work-image"></div>
+            <div class="left-block">
+                <h2 class="Tonic">Tonic</h2>
+                <div class="frame1">
+                    <p class="canopy">CANOPY</p>
+                    <p class="back-end-dev">Back End Dev</p>
+                    <p class="year">2015</p>
+                </div>
+                <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <a class="html">html</a>
+                    <a class="css">css</a>
+                    <a class="javascript">javascript</a>
+                </div>
+                <button class="button">See Project</button>
+            </div>
+
+        </div>
+        <div class="card-work-2">
+            <div class="work-image-2"></div>
+            <div class="left-block">
+                <p class="Tonic">Multi-Post <br> Stories</p>
+                <div class="frame1">
+                    <p class="canopy">CANOPY</p>
+                    <p class="back-end-dev">Back End Dev</p>
+                    <p class="year">2015</p>
+                </div>
+                <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <a class="html">html</a>
+                    <a class="css">css</a>
+                    <a class="javascript">javascript</a>
+                </div>
+                <button class="button">See Project</button>
+            </div>
+        </div>
+        <div class="card-work-2">
+            <div class="work-image-3"></div>
+            <div class="left-block">
+                <p class="Tonic">Tonic</p>
```

Comment:
- [ ] This issue mentioned by the other reviewer, has not been fixed. Please see the [comment here ](https://github.com/ernestkorir/my-portfolio/pull/1#discussion_r979197237)and implement.